src/components/Layout.less

```diff
@@ -0,0 +1,92 @@
+@import "~clever-components/dist/less/fonts";
+@import (reference) "~clever-components/dist/less/index";
+@import "~clever-components/dist/less/type-size";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  .flexbox;
+  .flex--direction--column;
+  .text--medium;
+  .text--line-height-3;
+  color: @neutral_black;
+  font-family: "Proxima Nova";
+  height: auto;
+  margin: 0;
+  min-height: 100vh;
+  padding: 0;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line selector-no-id */
+#__MAIN__ {
+  .flexbox;
+  .flex--direction--column;
+  .flex--grow;
+  min-height: 100vh;
+  width: 100%;
+}
+
+.Layout--mainContainer {
+  width: 100%;
+}
+
+h1 { &:extend(.text--gargantuan); }
+
+h2 {
+  .display--block;
+  .margin--y--s;
+  .text--semi-bold;
+  &:extend(.text--huge);
```

Comment:
We avoid using :extend pretty much everywhere. I should add not to use it to the styleguide.